mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,15 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.purgeOrder = async (req, res) => {
```

Comment:
The function definition is missing the rest of the implementation. It should be completed or a comment should be added explaining why it's incomplete.